README.md

```diff
@@ -1,2 +1,19 @@
 # Breakable-Toy-I-Todo-App
-Todo App designed in React with Java Spring Boot.
+## Description
+This application is intended to help users properly manage all the tasks they have to perform in their daily lives. In this way, it allows the creation, editing and deletion of tasks according to the user's needs; in the same way, a specific search for tasks can be carried out through filters by name, status or priority, and the list can also be sorted by priority and due date. Each task consists mainly of name, status and due date.
+
+Likewise, there is a section that shows the average time it takes a person to complete a task from the time it is created until it is marked as completed.
+
+### Technologies used
+- **Front-end:** Vite(React), TypeScript, Ant Design, Axios, Vitest, React-icons, Date-fns, Sonner and Redux-toolkit.
+- **Back-end:** Java, Spring Boot, Junit and Mockito.
+
+## Instalation
+1. Clone the repository:
+``` bash
+    git clone https://github.com/OmarRdz12/Breakable-Toy-I-Todo-App.git
```

Comment:
do the user needs to have java, node etc. Include the steps to have both apps running

Reply:
you can check https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/

Reply:
you can also reference the other two readmeds from here